app/controllers/course/survey/responses_controller.rb

```diff
@@ -28,7 +28,12 @@ def edit
   end
 
   def update
-    @response.submit if params[:response][:submit]
+    if params[:response][:submit]
+      @response.submit
+    elsif params[:response][:unsubmit]
+      @response.unsubmit
```

Comment:
we can filter out the params in strong params ( based on ability ), so for students this param should be filtered out.